webidl/MathMLLinkableElement.idl

```diff
@@ -0,0 +1,16 @@
+[Exposed=Window]
+interface MathMLLinkableElement : MathMLElement { 
+  [CEReactions] attribute DOMString target;
+  [CEReactions] attribute DOMString download;
+  [CEReactions] attribute USVString ping;
+  [CEReactions] attribute DOMString rel;
+  [SameObject, PutForwards=value] readonly attribute DOMTokenList relList;
+  [CEReactions] attribute DOMString hreflang;
+  [CEReactions] attribute DOMString type;
+
+  [CEReactions] attribute DOMString text;
```

Comment:
what is this? Alt text?

Reply:
from `a` in HTML.. 

> a . text
Same as textContent.